src/editors/containers/ProblemEditor/components/EditProblemView/SettingsWidget/hooks.js

```diff
@@ -207,10 +207,16 @@ export const typeRowHooks = ({
   updateAnswer,
 }) => {
   const onClick = () => {
+    // Dropdown problems can only have one correct answer. When there is more than one correct answer
+    // from a previous problem type, the correct attribute for selected answers need to be set to false.
     if (typeKey === ProblemTypeKeys.DROPDOWN) {
+      // Check that previous problem had greater than one correct answer
```

Comment:
This comment only restates what the line below says. If someone changes the code, they need to update the comment, and if the code is not clear than likely the comment is not clear. Better to only have the upper-level comment.

Reply:
```suggestion
```